tensorflow/lite/micro/kernels/conv_test_data.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_LITE_MICRO_KERNELS_CONV_TEST_DATA_H_
+#define TENSORFLOW_LITE_MICRO_KERNELS_CONV_TEST_DATA_H_
+
+#include "tensorflow/lite/c/common.h"
+
+namespace tflite {
+namespace testing {
+namespace conv {
+extern const int8_t kInput1x32x32x3[];
+extern const int8_t kFilter8x3x3x3[];
+extern const int32_t kBiasQuantized8[];
+extern const int8_t kGoldenOutput1x16x16x8[];
```

Comment:
Since we will likely have even more such arrays, I think we will want to come up with some naming convention to prevent ODR violations.

For example, kBiasQuantized8 is likely too generic a name.

Any ideas?